src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -41,7 +41,8 @@
     :image             :icon
     :image-props       :i/derivated-path
     :action            :button
-    :action-props      {:on-press    #(js/alert "Coming soon!")
+    :action-props      {:on-press    #(rf/dispatch [:navigate-to :screen/wallet.edit-derivation-path
```

Comment:
should we keep this feature flagged while it's in progress? 🤔

Reply:
Yes I will update before merging

Reply:
Nice! 🙏